app/backend/src/couchers/db.py

```diff
@@ -118,19 +118,39 @@ def new_password_reset_token(session, user, hours=2):
     return password_reset_token, f"{hours} hours"
 
 
-def set_email_change_token(session, user, hours=2):
+def set_email_change_tokens(user, confirm_with_both_emails, hours=2):
     """
-    Make a new email change token that's valid for `hours` hours for this user
+    If the user does not have a password, they need to confirm their email change via their old email in addition to their new email; 'confirm_with_both_emails' flags if confirmation via the old email is required
+
+    Make email change tokens which are valid for `hours` hours
 
     Note: does not call session.commit()
 
-    Returns token and expiry text
+    Returns two tokens and expiry text
     """
-    token = urlsafe_secure_token()
-    user.new_email_token = token
+    old_email_token = ""
```

Comment:
You can move this into the `else`; in python, `if` statements don't have their own scope

Reply:
Didn't know that. Cool